pyroclast/common/visualizable.py

```diff
@@ -98,3 +98,25 @@ def sensitivity_map(self, x, softmax=False):
                 result = tf.nn.softmax(result)
             grads = tape.gradient(result, x)
         return grads
+
+    def smooth_grad(self, x, n=50, sigma=0.01):
+        """Computes the smooth grad saliency-map
```

Comment:
Isn't "SmoothGrad" one word with specific capitalization? Further, if this implementation is based on a paper, you should cite it here.